src/main/steps/common/common.content.ts

```diff
@@ -123,8 +121,6 @@ const cy: typeof en = {
   wife: 'gwraig',
   partner: 'partner',
   civilPartner: 'partner sifil',
-  marriage: 'priodas',
-  civilPartnership: 'partneriaeth sifil',
```

Comment:
Did you mean to delete these translations?

Reply:
the property already exist

Reply:
[just a couple of lines above](https://github.com/hmcts/nfdiv-frontend/pull/296/files/8893afce7842c3cd874067fc250177090c333078#diff-fdd46b7ae6a19351332967acc233ae0022097bfd3f13fd879a434f2c52768617R118)

Reply:
As yes, must have come from 2 very similar PRs with the same/similar translations 👍